<launch>
	<include file="$(find VO_ORB)/launch/r200.launch" />
	<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf" output="screen">
      	<param name="output_frame" value="odom"/>
      	<param name="base_footprint_frame" value="base_link"/>
	  	<param name="freq" value="20.0"/>
	  	<param name="sensor_timeout" value="1.0"/>  
	  	<param name="odom_used" value="true"/>
	  	<param name="imu_used" value="true"/>
	  	<param name="vo_used" value="true"/>

        <!-- <remap from="/robot_pose_ekf/odom_combined" to="/odom" /> -->
        <remap from="/odom" to="/wheelodom" />
        <remap from="/vo" to="/ORB2_RGBD/visual_odom" />
    </node> 
</launch>